### ISSUE TITLE
After the update, the plugin is not loading.

### ISSUE THREAD

**Issue by v1k70rk4** [amaximus/bkk_stop#23]
Hello,

After I installed the latest version of HomeAssistant (https://www.home-assistant.io/blog/2023/06/a/release-20236/), the plugin throws the following error. I assume it's because of Python 3.11.

I would greatly appreciate it if you could look into whether anything can be done about it!

Thanks in advance for your work.

```
Logger: homeassistant.loader
Source: custom_components/bkk_stop/sensor.py:49
Integration: bkk_stop (documentation, issues)
First occurred: 22:39:35 (2 occurrences)
Last logged: 22:39:35

Unexpected exception importing platform custom_components.bkk_stop.sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 833, in get_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 850, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/bkk_stop/sensor.py", line 49, in <module>
    @asyncio.coroutine
     ^^^^^^^^^^^^^^^^^
AttributeError: module 'asyncio' has no attribute 'coroutine'
```

**Comment by amaximus** [comment]
This has been fixed in the newest release.

Please update the custom component on your instance and get back to me with results.

**Comment by v1k70rk4** [comment]
> This has been fixed in the newest release.
> 
> Please update the custom component on your instance and get back to me with results.


Hello, thank you for the quick help, but unfortunately, now this error has occurred instead:

```
Logger: homeassistant.components.sensor
Source: helpers/aiohttp_client.py:73
Integration: Érzékelő (documentation, issues)
First occurred: 12:22:03 (2 occurrences)
Last logged: 12:22:03

bkk_stop: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 537, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 784, in async_device_update
    await self.async_update()
  File "/config/custom_components/bkk_stop/sensor.py", line 164, in async_update
    self._bkkdata = await response.json()
                    ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/aiohttp_client.py", line 73, in json
    return await super().json(*args, loads=loads, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiohttp/client_reqrep.py", line 1104, in json
    raise ContentTypeError(
aiohttp.client_exceptions.ContentTypeError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain;charset=iso-8859-1', url=URL('https://go.bkk.hu/api/query/v1/ws/otp/api/where/arrivals-and-departures-for-stop.json?key=apaiary-test&version=3&appVersion=apiary-1.0&onlyDepartures=true&stopId=BKK_F02677&minutesAfter=25')
```

**Comment by amaximus** [comment]
Newly published version (2.7.0) addresses this issue.
You'll have to create an API key for yourself at https://opendata.bkk.hu to access BKK data and add it to the configuration as per the README.

Once you did the aboves, get back to me with results.

**Comment by v1k70rk4** [comment]
Thank you very much, it works. The key was instantly approved. Have a great day!
